README.md

```diff
@@ -36,14 +39,35 @@ in CI, but it's useful to run them locally too.
 $ make install-hooks
 ```
 
+### Environment Variables
+There is a file called `.env` that contains environment variables that are used by components that should behave differently in different environments.
```

Comment:
File is '.envrc' (not '.env')

Also not sure it's appropriate to include the .envrc in the repo - my preference would be to describe the environment variables in the documentation and let developers set them how they choose. (This is a taste thing though so up to you.)